designsafe/apps/workspace/templates/designsafe/apps/workspace/app_variant_plugin.html

```diff
@@ -0,0 +1,15 @@
+<section style="background-color: #f4f4f4; padding: 25px;">
+    <h3>Select a Version</h3>
+    {% for variant in listing %}
+    <article>
+        <div style="display:flex; align-items: center; justify-content: space-between;">
+            <h4>{{variant.label}}</h4>
+            <a href="#"><button class="btn btn-success">Get Started</button></a>
```

Comment:
We'll need to add an `href` field just for the plugin context for variants, that calculates the url based on the app id and version if defined. URLs at the moment will look like
```
/rw/workspace/applications/${APP_ID}?${APP_VERSION}
```

`APP_VERSION` could be undefined